nav2_costmap_2d/src/costmap_2d_node.cpp

```diff
@@ -43,13 +43,25 @@ int main(int argc, char ** argv)
 {
   rclcpp::init(argc, argv);
 
-  auto node = rclcpp::Node::make_shared("costmap_node");
   std::string name = "costmap";
   tf2_ros::Buffer buffer(tf2::durationFromSec(10));
   tf2_ros::TransformListener tf(buffer);
-  nav2_costmap_2d::Costmap2DROS lcr(name, buffer);
 
-  rclcpp::spin(node);
+  auto t_node = rclcpp::Node::make_shared("time_node");
+
+  geometry_msgs::msg::TransformStamped base_rel_map;
```

Comment:
Maybe add a comment to this section to explain that you're creating a static transform and why.

Reply:
I actually intended to remove this static transform completely (I thought I already had) from this PR.